internal/npm_package/packager.js

```diff
@@ -97,43 +97,49 @@ function main(args) {
 
   // src like baseDir/my/path is just copied to outDir/my/path
   for (src of srcsArg.split(',').filter(s => !!s)) {
-    if (!src.startsWith(baseDir)) {
-      throw new Error(`${src} in 'srcs' does not reside in the base directory, ` +
-        `generated file should belong in 'deps' instead.`);
+    if (src.startsWith('external/')) {
```

Comment:
unit tests for this file that could be updated?

Reply:
Good call. Spec & test updated to include external repo files in srcs & deps.

Had to use ts_library for this instead of mock_ts_library since behavior of the two differs slightly in file paths.